images/base/README.md

```diff
@@ -18,4 +18,4 @@ See also [`pkg/cluster`](./../../pkg/cluster) for logic that interacts with this
 
 ## Design
 
-See [docs/base-image.md](./../../docs/design/base-image.md) for more design details.
+See [docs/base-image.md](./../../site/content/docs/design/base-image.md) for more design details.
```

Comment:
can we do the same as the node image change below here, and for the other site links? :upside_down_face:

Reply:
my mistake. will update.